INSTALL.md

```diff
@@ -19,6 +19,8 @@ $ conda install -c pytorch -c nvidia faiss-gpu=1.8.0
 
 # GPU(+CPU) version with NVIDIA RAFT
 $ conda install -c pytorch -c nvidia -c rapidsai -c conda-forge faiss-gpu-raft=1.8.0
+
+# GPU(+CPU) version using ROCm not yet available
```

Comment:
1. Should we create an issue to track getting ROCm to be published to Conda? I assume that's the based on the comment.
2. nit: Can we cahnge `using ROCm` to `with AMD ROCm`? Here and elsewhere.

Reply:
1. I think that would be good for tracking progress/blockers/etc
2. Fixed with [0bbb4116212b1bf9dcee7afb11d52fb9569e0a1c](https://github.com/facebookresearch/faiss/pull/3856/commits/0bbb4116212b1bf9dcee7afb11d52fb9569e0a1c)